pkg/runner/playwright_test.go

```diff
@@ -0,0 +1,45 @@
+package runner
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+
+	cp "github.com/otiai10/copy"
+)
+
+func TestRun(t *testing.T) {
+	t.Skip("move this test to e2e test suite with valid environment setup")
```

Comment:
Would be nice to have some automated integration tests, is that possible to implement here?

Reply:
Can you give an example of the tests of other 'executors' that I need to implement?
I will try to implement it with reference to examples.

What features should I test?

Reply:
@vLia 
I already added examples.
It was configured by referring to "testkube-executor-cypress".

Need more tests?

Reply:
<img width="908" alt="image" src="https://user-images.githubusercontent.com/7044743/213377493-1cc74447-c221-4dc1-95fd-2cb3863f188a.png">

Reply:
Looks great now, it works on my machine as well 👍